tests/registry_disk_test.go

```diff
@@ -0,0 +1,103 @@
+/*
+ * This file is part of the kubevirt project
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * Copyright 2017 Red Hat, Inc.
+ *
+ */
+
+package tests_test
+
+import (
+	"flag"
+	"strings"
+
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+
+	"k8s.io/apimachinery/pkg/runtime"
+
+	"kubevirt.io/kubevirt/pkg/api/v1"
+	"kubevirt.io/kubevirt/pkg/kubecli"
+	"kubevirt.io/kubevirt/pkg/virt-controller/services"
+	"kubevirt.io/kubevirt/tests"
+)
+
+var _ = Describe("RegistryDisk", func() {
+
+	flag.Parse()
+
+	coreClient, err := kubecli.Get()
+	tests.PanicOnError(err)
+
+	restClient, err := kubecli.GetRESTClient()
+	tests.PanicOnError(err)
+
+	BeforeEach(func() {
+		tests.BeforeTestCleanup()
+	})
+
+	LaunchVM := func(vm *v1.VM) runtime.Object {
+		obj, err := restClient.Post().Resource("vms").Namespace(tests.NamespaceTestDefault).Body(vm).Do().Get()
```

Comment:
I agree there's room for improvement here. Part of the reason I didn't start using the kubevirt client is because helper functions expect the runtime obj to be passed in. 

I've noted to follow up on this in a separate patch. To do it right, this is going to involve a more broad set of changes to our functional tests. The code is the way it is in the registry disk tests because it is consistent with the other functional tests and the common utils.go file.